validator/src/commands/wait_for_restart_window/mod.rs

```diff
@@ -341,3 +363,104 @@ pub fn wait_for_restart_window(
     println!("{}", style("Ready to restart").green());
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use {super::*, crate::commands::tests::verify_args_struct_by_command, std::str::FromStr};
+
+    #[test]
+    fn verify_args_struct_by_command_wait_for_restart_window_default() {
+        verify_args_struct_by_command(
+            command(&DefaultArgs::default()),
+            vec![COMMAND],
+            WaitForRestartWindowArgs {
+                min_idle_time: 10,
+                identity: None,
+                max_delinquent_stake: 5,
+                skip_new_snapshot_check: false,
+                skip_health_check: false,
```

Comment:
These appear to be the default values for each arg. Supposing one of these changed (ie `min_idle_time` from 10 to 5), it would be annoying to have to update each individual unit test with the new value

How about we implement a `Default` trait for `WaitForRestartWindowArgs` within the `tests` module. This unit test can check:
```rust
vec![COMMAND],
WaitForRestartWindowArgs::default()
```
And then other tests such as the `skip-new-snapshot-check` can use:
```
vec![COMMAND, "--skip-new-snapshot-check"],
WaitForRestartWindowArgs {
    skip_new_snapshot_check: true,
    ..WaitForRestartWindowArgs::default()   
}
```

Reply:
yeah, I have a plan to refactor the default args. I thought about doing it in some separate PRs, but maybe it's fine to do it all together to speed things up haha :trollface: 

338c71a

Reply:
Yeah, I'm fine with combining it into single PR as long as the commits are well structured (like you have done in the past)